packages/v4/src/content/design-guidelines/components/truncate/truncate.md

```diff
@@ -0,0 +1,47 @@
+---
+id: Truncate design guidelines
+section: components
+---
+
+A **truncate** is a tool used to shorten numeric and non-numeric character strings, typically when the string overflows its container. 
+
+## Elements
```

Comment:
Would bold the first words for each of these:
eg:
**Content:** text string to truncate